registry/base_registry.go

```diff
@@ -136,6 +136,13 @@ func (r *BaseRegistry) Register(conf *common.URL) error {
 		ok  bool
 		err error
 	)
+	// if developer define registry port and ip, use it first.
+	if ipToRegistry := os.Getenv("DUBBO_IP_TO_REGISTRY"); ipToRegistry != "" {
```

Comment:
Why are ip and port from environment instead of configuration?

Reply:
Because this feature is supported in dubbo-java, and I use this feature in design of docker example

Reply:
ok